action/remove.go

```diff
@@ -33,7 +33,8 @@ var kubeGet kubeGetter = func(m string) string {
 func Remove(chart, homedir string, force bool) {
 	chartPath := filepath.Join(homedir, WorkspaceChartPath, chart)
 	if _, err := os.Stat(chartPath); err != nil {
-		log.Die("Chart not found. %s", err)
+		log.Err("Chart not found. %s", err)
+		return
```

Comment:
@sgoings `log.Die` as previously used there. It performs an `os.Exit(1)` ending the applications execution. When I switched to `log.Err` it no longer exited the application. The `return` ends the execution of the `Action` handler.